doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
@sam-github That would be semver-minor then, if it would not have the deprecation notice =).